code/game/objects/items/weapons/grenades/ghettobomb.dm

```diff
@@ -57,6 +121,18 @@
 /obj/item/weapon/grenade/iedcasing/prime() //Blowing that can up
 	update_mob()
 	explosion(src.loc,0,0,2,2) //explosion(src.loc,-1,-1,-1, flame_range = range)
+	if(istype(loc, /obj/item/weapon/legcuffs/beartrap))
+		var/obj/item/weapon/legcuffs/beartrap/boomtrap = loc
+		if(istype(boomtrap.loc, /mob/living/carbon/human))
+			var/mob/living/carbon/human/H = loc.loc
+			if(H.legcuffed)
+				var/datum/organ/external/l = H.get_organ("l_leg")
+				var/datum/organ/external/r = H.get_organ("r_leg")
+				if(l && !(l.status & ORGAN_DESTROYED))
+					l.status |= ORGAN_DESTROYED
+				if(r && !(r.status & ORGAN_DESTROYED))
+					r.status |= ORGAN_DESTROYED
+				qdel(H.legcuffed)
```

Comment:
Just a suggestion, this is insanely powerful for just a bear trap+IED---the janitor's *traitor* item blows legs off guarnateed--perhaps it should just trigger the 0,0,2 explosion instantly? That would still be a decent landmine without a guaranteed chance blow off both legs every time.